src/data/sql-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const sqlQuiz = [
+  {
+    Question:
+      "In SQL, what keyword is used if the WHERE clause fails when being used with aggregate functions?",
+    Answer: "HAVING",
+    Distractor1: "ADD",
+    Distractor2: "DISTINTC",
+    Distractor3: "LIMIT",
+    Explanation:
+      "With the HAVING clause, you can arrange the data in your database into many groups when you use it with the GROUP BY keyword.",
```

Comment:
I think this revised explanation works better with your question

```suggestion
      "In SQL, you will have to use the HAVING clause because the WHERE clause fails when used with aggregate functions.",
```